client/state/data-layer/wpcom/read/following/mine/index.js

```diff
@@ -19,6 +19,13 @@ import { errorNotice } from 'state/notices/actions';
 const ITEMS_PER_PAGE = 200;
 const MAX_ITEMS = 2000;
 
+function toNumberOrUndefined( val ) {
+	if ( val ) {
+		return Number( val );
+	}
+	return undefined;
+}
```

Comment:
> fun cases

one more constraint, number or string

```js
const toNumberOrUndefined = val =>
	val && ( 'string' === typeof val || 'number' === typeof val ) && _.isFinite( Number( val ) )
		? Number( val )
		: undefined
```

> why does everyone hate the function keyword? 😄

in all seriousness, because of `this`